eos-tech-support/eos-write-live-image

```diff
@@ -484,6 +491,17 @@ if [ "$EXPAND" ]; then
     fi
 fi
 
+if [ "$PERSISTENT" ]; then
+    # we need 512 byte alignment for device mapper, so let's do this in KBytes.
+    FREE_SPACE_KBYTES=$(df -P -B1 "$DIR_IMAGES_ENDLESS" | awk 'NR==2 {print $4}')
+    let FREE_SPACE_KBYTES=FREE_SPACE_KBYTES/1024
```

Comment:
Wow, I have *never* seen `let` before. I have always written this kind of thing as:
```bash
FREE_SPACE_KBYTES=$(( FREE_SPACE_KBYTES / 1024 ))
```